specs/altair/beacon-chain.md

```diff
@@ -523,26 +529,28 @@ def process_sync_committee(state: BeaconState, aggregate: SyncAggregate) -> None
     # Verify sync committee aggregate signature signing over the previous slot block root
     previous_slot = Slot(max(int(state.slot), 1) - 1)
     committee_indices = get_sync_committee_indices(state, get_current_epoch(state))
-    participant_indices = [index for index, bit in zip(committee_indices, aggregate.sync_committee_bits) if bit]
+    included_indices = [index for index, bit in zip(committee_indices, aggregate.sync_committee_bits) if bit]
     committee_pubkeys = state.current_sync_committee.pubkeys
-    participant_pubkeys = [pubkey for pubkey, bit in zip(committee_pubkeys, aggregate.sync_committee_bits) if bit]
+    included_pubkeys = [pubkey for pubkey, bit in zip(committee_pubkeys, aggregate.sync_committee_bits) if bit]
     domain = get_domain(state, DOMAIN_SYNC_COMMITTEE, compute_epoch_at_slot(previous_slot))
     signing_root = compute_signing_root(get_block_root_at_slot(state, previous_slot), domain)
-    assert eth2_fast_aggregate_verify(participant_pubkeys, signing_root, aggregate.sync_committee_signature)
-
-    # Reward sync committee participants
-    proposer_rewards = Gwei(0)
-    active_validator_count = uint64(len(get_active_validator_indices(state, get_current_epoch(state))))
-    for participant_index in participant_indices:
-        proposer_reward = get_proposer_reward(state, participant_index)
-        proposer_rewards += proposer_reward
-        base_reward = get_base_reward(state, participant_index)
-        max_participant_reward = base_reward - proposer_reward
-        reward = Gwei(max_participant_reward * active_validator_count // (len(committee_indices) * SLOTS_PER_EPOCH))
-        increase_balance(state, participant_index, reward)
-
-    # Reward beacon proposer
-    increase_balance(state, get_beacon_proposer_index(state), proposer_rewards)
+    assert eth2_fast_aggregate_verify(included_pubkeys, signing_root, aggregate.sync_committee_signature)
+
+    # Compute the maximum sync rewards for the slot
+    total_active_increments = get_total_active_balance(state) // EFFECTIVE_BALANCE_INCREMENT
+    total_base_rewards = Gwei(get_base_reward_per_increment(state) * total_active_increments)
```

Comment:
`total_active_increments = 4M validators max, 32 increments each = 32*4000000 = 128000000`, `log2(128000000) = 26.93... bits`. There were 28 bits of room. Ok...